### PR TITLE
Remove unreachable branch in `test_rai_insights_save_and_load_scenarios.py`

### PR DESCRIPTION
**Pull request by gaugup**
## Description

The pytest parameter list does not contain any other manager name.

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: responsibleai/tests/rai_insights/test_rai_insights_save_and_load_scenarios.py [hunk 1]
```diff
@@ -207,11 +207,8 @@ def test_rai_insights_add_save_load_save(self, manager_type, adult_data):
             )
         elif manager_type == ManagerNames.ERROR_ANALYSIS:
             rai_insights.error_analysis.add()
-        elif manager_type == ManagerNames.EXPLAINER:
```
Comment (Advitya17) [inline responsibleai/tests/rai_insights/test_rai_insights_save_and_load_scenarios.py:210]: Nit: Why is this change needed? Won't the value error double check that an undesired enum doesn't go through, and won't we be assuming with the `else` that we have to add in the explainer? Perhaps not a big deal coz this is a testing file.

**Review by Advitya17 — APPROVED** [review]

**Review by imatiach-msft — APPROVED** [review]

File: responsibleai/tests/rai_insights/test_rai_insights_save_and_load_scenarios.py (hunk 1 above)
Comment (romanlutz) [reply responsibleai/tests/rai_insights/test_rai_insights_save_and_load_scenarios.py:210]: Would also like to know :-)

Comment (gaugup) [reply responsibleai/tests/rai_insights/test_rai_insights_save_and_load_scenarios.py:210]: The new manager would be added anyways to the parameter list and the code to handle it would be added in the same PR, isn't it? It is dead code anyways.
